source/frontend/src/models/api/generated/ApiGen_Concepts_Lease.ts

```diff
@@ -30,23 +27,21 @@ export interface ApiGen_Concepts_Lease extends ApiGen_Base_BaseAudit {
   otherType: string | null;
   expiryDate: string | null;
   startDate: string;
-  renewalDate: string | null;
   renewalCount: number;
   paymentReceivableType: ApiGen_Base_CodeType<string> | null;
   type: ApiGen_Base_CodeType<string> | null;
   initiatorType: ApiGen_Base_CodeType<string> | null;
   responsibilityType: ApiGen_Base_CodeType<string> | null;
   categoryType: ApiGen_Base_CodeType<string> | null;
   purposeType: ApiGen_Base_CodeType<string> | null;
-  statusType: ApiGen_Base_CodeType<string> | null;
   region: ApiGen_Base_CodeType<number> | null;
   programType: ApiGen_Base_CodeType<string> | null;
   returnNotes: string | null;
   responsibilityEffectiveDate: string | null;
-  properties: ApiGen_Concepts_PropertyLease[] | null;
-  securityDeposits: ApiGen_Concepts_SecurityDeposit[] | null;
+  fileProperties: ApiGen_Concepts_PropertyLease[] | null;
```

Comment:
this change seems to be the cause of lease properties not working as well as the hard crashes.